migrations.yml

```diff
@@ -1,4 +1,4 @@
 name: ChoreDo Migrations
 migrations_namespace: Choredo\Migrations
 table_name: doctrine_migration_versions
-migrations_directory: migrations
\ No newline at end of file
+migrations_directory: database/migrations
```

Comment:
Is this right? `/migrations` is the folder on my local.

Reply:
Might be nice if we end up with fixtures or w/e